source/dub/dependencyresolver.d

```diff
@@ -53,15 +53,27 @@ class DependencyResolver(CONFIGS, CONFIG) {
 		}
 	}
 
+	static struct ConfigEntry
+	{
+		static struct Depender
+		{
+			TreeNode origin;
+			TreeNodes dependency;
+		}
+
+		CONFIG[] all_configs;
+		bool any_config;
```

Comment:
Naming convention: `allConfigs`, `anyConfig`.

Reply:
I adapted to the original naming conventions of the file which used exactly such names